schema_poc/create_plate_map_sample_v1.avsc

```diff
@@ -0,0 +1,89 @@
+{
+  "namespace": "uk.ac.sanger.psd",
+  "type": "record",
+  "name": "CreatePlateMapMessage",
+  "doc": "A create message to process specified plate map data.",
+  "fields": [
+    {
+      "name": "messageUuid", "type": "string",
+      "doc": "Unique message ID."
+    },
+    {
+      "name": "messageCreateDateUtc",
+      "doc": "Date (UTC) that the message was created.",
+      "type": {
+        "type": "long",
+        "logicalType": "timestamp-millis"
+      }
+    },
+    {
+      "name": "plate",
+      "doc": "The plate to process.",
+      "type": {
+        "name": "PlateMap", "type": "record",
+        "doc": "Plate map data.",
+        "fields": [
+          {
+            "name": "labId", "type": "string",
+            "doc": "ID for the lighthouse where the plate originated."
+          },
+          {
+            "name": "plateBarcode", "type": "string",
+            "doc": "The barcode for this plate."
+          },
+          {
+            "name": "samples",
+            "doc": "An array of data for samples on the plate.",
+            "type": {
+              "type": "array",
+              "items": {
+                "name": "PlateMapSample", "type": "record",
+                "doc": "A plate map sample.",
+                "fields": [
+                  {
+                    "name": "sampleUuid", "type": "string",
+                    "doc": "Unique sample ID, assigned by PAM."
+                  },
+                  {
+                    "name": "rootSampleId", "type": "string",
+                    "doc": "Internal ID for the sample from the lighthouse."
+                  },
```

Comment:
rnaId will be sample name in SS, I will also vote for receiving it even if it is <plateBarcode>_<PlateCoordinate>

Reply:
I would agree completely if it wasnt because we have to create the sample with the rna_id as the sample name; without it we will need to recreate it again when creating the sample in SS by concatenating with '_'  and coordinates could be also be unpadded so we may not generate the rna_id as it was originally.

Reply:
(I haven't been involved in these discussions at all, so feel free to ignore me, just noticed this conversation)
I would vote for keeping the sample name in the same format going forward - the less that changes, the fewer headaches for reporting and data fixes in the future I think.